rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1321,6 +1321,52 @@ def test_upgrade_db_22_to_23_without_frontend_settings(data_dir, user_data_dir):
     assert db.get_version() == 23
 
 
+def test_upgrade_db_23_to_24(user_data_dir):  # pylint: disable=unused-argument
+    """Test upgrading the DB from version 23 to version 24.
+
+    Deletes the AdEx used query ranges, drops the AdEx events table and re-creates it.
+    """
+    msg_aggregator = MessagesAggregator()
+    _use_prepared_db(user_data_dir, 'v23_rotkehlchen.db')
+    db = _init_db_with_target_version(
+        target_version=24,
+        user_data_dir=user_data_dir,
+        msg_aggregator=msg_aggregator,
+    )
+    cursor = db.conn.cursor()
+    # Make sure AdEx used query ranges have been deleted
+    assert cursor.execute(
+        'SELECT COUNT(*) from used_query_ranges WHERE name = "adex_events";',
```

Comment:
This is wrong as I mentioned above. I would advice you to check BEFORE the upgrade in the test that both in the events and in the used query ranges there is data.